### ISSUE TITLE
Shopping Cart Dropdown Content

### ISSUE THREAD

**Issue by ohnno** [ohnno/Brosa#31]
Location: Top Navigation
Browser: Firefox
OS: OS X Mavericks
Screen: Laptop

Current shopping cart dropdown content/layout does not make sense.

**Comment by davidywei** [comment]
This will make sense after new products are uploaded (instead of the current test ones) - no tech work required here.

**Comment by ohnno** [comment]
Yep cool, I'll just leave the issue open until the products are uploaded and it's resolved on the website.
